### ISSUE TITLE
Numbers of ticks and number of holidays

### ISSUE THREAD

**Issue by PeterTymms** [DurhamARC/classroom-abm#10]
If the number of weeks per holiday is inceased then the number of ticks overall increases at the end of a run. But I would expected the number of ticks to remain constant.
Also  home learning should only occur when the kids are not at school

**Comment by alisonrclarke** [comment]
The current spec document for timings says:

> The weekends and holidays should be added to the current number of ticks.

which is why it works this way. I realise that is naive when altering the length of holidays however, and especially won't work for long periods of home learning! I'll change it so that the number of ticks does not depend on the length of holidays. 

@PeterTymms Should we go for a total of 300 days, or 317 (i.e. number of days from 1st September to 15th July?)

**Comment by alisonrclarke** [comment]
> Also home learning should only occur when the kids are not at school

I recently made the change following a discussion with you, where you said there should be some time after school where there was passive learning, so now there is learning in school (active + passive) for 5.5h and after school (passive) for 5.5h. Previous to that we just had 5.5h of active + passive learning on school days, and 5.5h of passive learning on holidays/weekends. Please can you clarify how many hours of each type of learning there should be on schooldays vs holidays?

**Comment by PeterTymms** [comment]
Thanks Alison,  I think that 317 days would be best so as to match real ife as closely as possible   Sent from Mail for Windows 10 From: Alison ClarkeSent: 03 November 2020 11:22To: DurhamARC/classroom-abmCc: PeterTymms; MentionSubject: Re: [DurhamARC/classroom-abm] Numbers of ticks and number of holidays (#10) The current spec document for timings says:The weekends and holidays should be added to the current number of ticks.which is why it works this way. I realise that is naive when altering the length of holidays however, and especially won't work for long periods of home learning! I'll change it so that the number of ticks does not depend on the length of holidays.@PeterTymms Should we go for a total of 300 days, or 317 (i.e. number of days from 1st September to 15th July?)—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by PeterTymms** [comment]
I see that my mind is changing as I run the models. I think that  One a school day we should have 5.5h of school learning which involves no passive learning; plus 5.5h of passive learning corresponding to out of school time On holidays we should have 11 hrs of passive learning. Hope this makes sense and sounds sensible  Sent from Mail for Windows 10 From: Alison ClarkeSent: 03 November 2020 11:26To: DurhamARC/classroom-abmCc: PeterTymms; MentionSubject: Re: [DurhamARC/classroom-abm] Numbers of ticks and number of holidays (#10) Also home learning should only occur when the kids are not at schoolI recently made the change following a discussion with you, where you said there should be some time after school where there was passive learning, so now there is learning in school (active + passive) for 5.5h and after school (passive) for 5.5h. Previous to that we just had 5.5h of active + passive learning on school days, and 5.5h of passive learning on holidays/weekends. Please can you clarify how many hours of each type of learning there should be on schooldays vs holidays?—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by PeterTymms** [comment]
Works for me

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: TYMMS, PETER B. <peter.tymms@gmail.com>
Sent: Tuesday, November 3, 2020 11:52:50 AM
To: DurhamARC/classroom-abm <reply@reply.github.com>; DurhamARC/classroom-abm <classroom-abm@noreply.github.com>
Cc: Mention <mention@noreply.github.com>; BROWN, CHRIS D. <chris.brown@durham.ac.uk>
Subject: RE: [DurhamARC/classroom-abm] Numbers of ticks and number of holidays (#10)


[EXTERNAL EMAIL] Do not open links or attachments unless you recognise the sender and know the content is safe. Otherwise, use the Report Message button or report to phishing@durham.ac.uk.

I see that my mind is changing as I run the models. I think that



One a school day we should have 5.5h of school learning which involves no passive learning; plus 5.5h of passive learning corresponding to out of school time



On holidays we should have 11 hrs of passive learning.



Hope this makes sense and sounds sensible



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by alisonrclarke** [comment]
> On a school day we should have 5.5h of school learning which involves no passive learning; plus 5.5h of passive learning corresponding to out of school time On holidays we should have 11 hrs of passive learning.

That makes sense. I'll open up another issue to keep this separate from the issue about total number of ticks and holidays.

**Comment by PeterTymms** [comment]
Thanks Sent from Mail for Windows 10 From: Alison ClarkeSent: 03 November 2020 12:01To: DurhamARC/classroom-abmCc: PeterTymms; MentionSubject: Re: [DurhamARC/classroom-abm] Numbers of ticks and number of holidays (#10) On a school day we should have 5.5h of school learning which involves no passive learning; plus 5.5h of passive learning corresponding to out of school time On holidays we should have 11 hrs of passive learning.That makes sense. I'll open up another issue to keep this separate from the issue about total number of ticks and holidays.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by PeterTymms** [comment]
I hope that I have succesfully added three files to a branch - one is code and the two others are input files
